### PR TITLE
DOCSP-38521: c language landing page

### PR DESCRIPTION
**Pull request by rustagir**
https://jira.mongodb.org/browse/DOCSP-38521

https://preview-mongodbrustagir.gatsbyjs.io/landing/DOCSP-38521-c-lang-langing/languages/c/

### CODE REVIEW

**Review by caitlindavey — APPROVED** [review]
Added on comment, but otherwise LGTM

**Comment by caitlindavey** [comment]
For the More Ways to Learn category, please change wording to "Learn about C and MongoDB"

File: source/languages/c.txt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   .. card::
+      :headline: View C developer resources
+      :url: https://www.mongodb.com/developer/languages/c/
+      :icon: /images/technical-mdb-devhub.svg
+      :icon-alt: DevHub icon
+
+      C Resources  →
+
+   .. card::
+      :headline: Take the Getting Started with C course
```
Comment (caitlindavey) [inline source/languages/c.txt:66]: Change to: Learn

Comment (rustagir) [reply source/languages/c.txt:66]: pushing this change on PR #173

Comment (rustagir) [reply source/languages/c.txt:66]: Accidentally merged this one

**Pull request merged**